src/cluster/src/lib.rs

```diff
@@ -10,8 +10,8 @@
 //!
 //! ```no_run
 //! use fluvio_cluster::ClusterInstaller;
-//! let installer = ClusterInstaller::new().build().unwrap();
-//! fluvio_future::task::run_block_on(installer.install_fluvio()).unwrap();
+//! let installer = ClusterInstaller::new().unwrap();
+//! async_std::task::block_on(installer.install_fluvio()).unwrap();
```

Comment:
unneeded use of block_on

Reply:
`install_fluvio` is async

Reply:
is this example or dockets?  if it is example, there is no need block on.   it should be just express as async snippet

Reply:
This is a doc test that is marked as `no_run`. So it will do a compile-check, but not run anything. I included the `async_std` line so that even users who don't really know much about async will know that you need to do that in order to get this to run.